README.md

```diff
@@ -197,6 +197,30 @@ describe Person do
 end
 ```
 
+### RSpec-Expectations
+
+Use this instead of the RSpec ifself in case you just use the specific module gem
+'rspec-expectations' to test webpart(s).
+
+Include `shoulda-matchers` in your Gemfile:
+
+``` ruby
+group :test do
+  gem 'shoulda-matchers'
+end
+```
+
+Then, configure the gem to integrate with RSpec specifying the test framework as `:rspec_exp`:
+
+    Shoulda::Matchers.configure do |config|
+      config.integrate do |with|
+        with.test_framework :rspec_exp
```

Comment:
I think it would be more expressive / more obvious to use `rspec_expectations` instead of `rspec_exp`.

Reply:
could the `exp` statement be understood some else than `expectations` in context of rspec?

Reply:
Going along with the use case, instead of `with.test_framework :rspec_exp`, this would be `with.test_framework :cucumber`.